src/Runner.jl

```diff
@@ -322,6 +331,13 @@ function generate_compiler_wrappers!(platform::AbstractPlatform; bin_path::Abstr
             push!(flags, "-D_GLIBCXX_USE_CXX11_ABI=0")
         end
 
+        # Simulate some of the `__OSX_AVAILABLE()` macro usage that is broken in GCC
+        # Currently, we only target 10.10, but eventually, we'll want to tailor this to `os_version(p)`
+        if 14 < 16
```

Comment:
huh?

Reply:
Eventually, this will be `if os_version(p) < 16`, but since that doesn't work just yet, I'm manually inlining `os_version(p)` as `14`.

Reply:
I think this is inlining too much, this is set also for non-macOS platforms